src/lib/EventPublisher.js

```diff
@@ -59,6 +74,11 @@ class EventPublisher extends EventEmitter {
 
     this._emitWithPrefix(eventData.name, event);
     this.emit(ALL_EVENTS, event);
+
+    // broadcast events only from the current worker
+    if (!event.fromMaster && cluster.isWorker) {
```

Comment:
I can use this either your variant, or mine. but it doesn't change the fact that I need something for webhookManagers. here we just prevent multiple rebroadcasting the same events. We prevent mirroring. I'll try to explain step by step:
1) worker 1: get event from device
2) worker 1: publish the event for itself and _send it to master_
3) worker 1: in webhookManager(1), get the event and do webhook job
4) master: get the event and broadcast it to all other workers
5) worker 2,3,4: get the event from master and _publish it to itself_ only, we prevent mirroring with fromMaster check or another way.doesn't matter
6) worker 2,3,4: own webhookManagers(2,3,4) filter the event and do nothing. They need to know that the event comes from master, but not from device that connects to the worker.